addons/purchase_requisition/tests/test_purchase_requisition.py

```diff
@@ -485,5 +469,40 @@ def test_08_purchase_requisition_sequence(self):
             'company_id': new_company.id,
         })
         self.bo_requisition.company_id = new_company
+        self.bo_requisition.line_ids.write({'price_unit': 10.0})
         self.bo_requisition.action_in_progress()
         self.assertTrue(self.bo_requisition.name.startswith("REQ_"))
+
+    def test_09_purchase_requisition_purchase_template(self):
+        quantity = 2
+        self.res_partner_2 = self.env['res.partner'].create({
+            'name': 'Azure Interior',
+        })
+
+        self.supplierinfo10 = self.env['product.supplierinfo'].create({
+            'partner_id': self.res_partner_2.id,
+            'product_id': self.product_09.id,
+            'price': 15.0,
+            'min_qty': quantity,
+        })
+
+        # Create a pruchase requisition with type purchase template and two product
+        line1 = (0, 0, {'product_id': self.product_09.id, 'product_uom_id': self.product_uom_id.id})
+        line2 = (0, 0, {'product_id': self.product_13.id, 'product_uom_id': self.product_uom_id.id})
```

Comment:
Command (`from odoo import Command`) is the preferred way to create lines now for readability (for all supported versions of Odoo at this point):

https://www.odoo.com/documentation/master/developer/reference/backend/orm.html?highlight=command#odoo.fields.Command

Also no need for the description comment since it's clear what is being done in this case

```suggestion
        line1 = Command.create({'product_id': self.product_09.id, 'product_uom_id': self.product_uom_id.id})
        line2 = Command.create({'product_id': self.product_13.id, 'product_uom_id': self.product_uom_id.id})
```